src/core/service.c

```diff
@@ -710,6 +710,9 @@ static int service_verify(Service *s) {
         if (s->type == SERVICE_DBUS && !s->bus_name)
                 return log_unit_error_errno(UNIT(s), SYNTHETIC_ERRNO(ENOEXEC), "Service is of type D-Bus but no D-Bus service name has been specified. Refusing.");
 
+        if (s->type == SERVICE_FORKING && exec_needs_pid_namespace(&s->exec_context))
+                return log_unit_error_errno(UNIT(s), SYNTHETIC_ERRNO(ENOEXEC), "Service of type Forking does not support PrivatePIDs=yes. Refusing.");
```

Comment:
Why uppercase?

Reply:
The rest of the logs in service_verify are like that e.g.
```
Service has Restart= set to either always or on-success, which isn't allowed for Type=oneshot services. Refusing.
```

Reply:
(sorry for the confusion, I was referring to "forking")

Reply:
oh yes this should be `Type=forking`, will fix